scripts/zones/Arrapago_Remnants/mobs/Qiqirn_Astrologer.lua

```diff
@@ -0,0 +1,86 @@
+-----------------------------------
+-- MOB: Qiqirn Astrologer
+-- Area: Arrapago Remnants
+-----------------------------------
+require("scripts/globals/monstertpmoves")
+require("scripts/globals/status")
+require("scripts/globals/msg")
+require("scripts/globals/teleports")
+require("scripts/globals/pathfind")
+local ID = require("scripts/zones/Arrapago_Remnants/IDs")
+-----------------------------------
+
+function onMobSpawn(mob)
+   mob:setMobMod(dsp.mobMod.HP_STANDBACK, -1) 
+end
+
+
+
+function onMobDisengage(mob)
+    
+    local run = mob:getLocalVar("run")
+    local instance = mob:getInstance()
+    local stage = instance:getStage()
+    local prog = instance:getProgress()
+    
+    if run == 1 then
+        mob:pathThrough(ID.points[stage][prog-1].point1, 9)
+        mob:setLocalVar("run", 2)
+    elseif run == 2 then
+        mob:pathThrough(ID.points[stage][prog-1].point2, 9)
+        mob:setLocalVar("run", 3)
+    elseif run == 3 then
+        mob:pathThrough(ID.points[stage][prog-1].point3, 9)
+        mob:setLocalVar("run", 4)
+    elseif run == 4 then
+        mob:pathThrough(ID.points[stage][prog-1].point4, 9)
+        mob:setLocalVar("run", 5)
+    elseif run == 5 then
+        mob:pathThrough(ID.points[stage][prog-1].point5, 9)
+        mob:setLocalVar("run", 6)
+    elseif run == 6 then
+        mob:pathThrough(ID.points[stage][prog-1].point6, 9)
+        mob:setLocalVar("run", 7)
+    end
```

Comment:
If the mod is disengaged on run == 7 should it be moving back to point1/run = 1?

Reply:
This... might have been a relic from when I was having an issue with pathing... Prior to a change I made, you could interrupt the mob before it reached the end of its route, and could technically prevent the mob from ever reaching the teleporter and warping away. I'll have to see if this is still an issue and necessary. Might be able to just remove it.

Reply:
I misunderstood your question at first... iirc the qiqirn's would maintain their position in their path regardless if they were claimed or not.